definitions/node.d.ts

```diff
@@ -1,21 +1,29 @@
-// Type definitions for Node.js v0.10.1
+// Type definitions for Node.js v0.12.0
```

Comment:
It'd be good if you could get someone from the build team to make these sort of changes for you, as they have the skills to sign off on this. Otherwise, you need to get one of them to at least sign off on this change. Either way - package it separately.